net.sapples.LiveCaptions.json

```diff
@@ -0,0 +1,108 @@
+{
+    "app-id" : "net.sapples.LiveCaptions",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "livecaptions",
+    "finish-args" : [
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--socket=fallback-x11",
+        "--env=LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/app/lib"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/onnx_root",
+        "*.la",
+        "*.a"
+    ],
+    "build-options" : {
+        "env" : {
+            "ONNX_ROOT" : "/app/onnx_root"
+        }
+    },
+    "modules" : [
+        {
+            "name" : "onnxruntime-x64",
+            "only-arches" : ["x86_64"],
+            "buildsystem" : "simple",
+            "sources" : [
+                {
+                    "type" : "file",
+                    "dest-filename" : "onnxruntime-linux-x64-1.13.1.tgz",
+                    "url" : "https://github.com/microsoft/onnxruntime/releases/download/v1.13.1/onnxruntime-linux-x64-1.13.1.tgz",
+                    "sha256" : "2c7fdcfa8131b52167b1870747758cb24265952eba975318a67cc840c04ca73e"
+                }
+            ],
+            "build-commands" : [
+                "tar -xvzf onnxruntime-linux-x64-1.13.1.tgz",
+                "mkdir -p $ONNX_ROOT",
+                "mv onnxruntime-linux-x64-1.13.1/* $ONNX_ROOT",
+
+                "mkdir -p $FLATPAK_DEST/lib",
+                "cp $ONNX_ROOT/lib/* $FLATPAK_DEST/lib"
```

Comment:
I don't understand why you copy eveyrthing to $ONNX_ROOT, but then copy again elsewhere and have it in the cleanup rules. Looks like it could be down to one single line.

Reply:
Include files located in `$ONNX_ROOT/include` are necessary during the livecaptions build process